packages/indicators/src/Builder/ArithmeticBuilder/ArithmeticBuilder.ts

```diff
@@ -102,41 +120,38 @@ export class ArithmeticBuilder extends Builder {
   private getVariables = () => Object.keys(this.config.aggregation);
 
   /**
-   * We use the provided analytics repo (pre-populated ) and builders for nested indicators
+   * We use the provided analytics repo and builders for nested indicators
    * to build analytics for the following categories of variables included in the formula:
    *
    * a. Parameters (they take precedence over other elements with clashing codes)
    * b. Nested indicators
    * c. "Primitive" elements (eg `dhis`, `tupaia` elements)
    */
   private buildAggregatedAnalytics = (
-    populatedAnalyticsRepo: AnalyticsRepository,
+    analyticsRepo: AnalyticsRepository,
     buildersByIndicator: Record<string, Builder>,
-    fetchOptions: FetchOptions,
+    extraAggregations: Aggregation[],
   ) =>
     this.getVariables()
       .map(variable => {
-        const analytics = this.getAnalyticsForVariable(
+        const aggregationList = [...this.config.aggregation[variable], ...extraAggregations];
```

Comment:
We need to add them here so that `fetchOptions` are correctly adjusted in [AnalyticsRepository](https://github.com/beyondessential/tupaia/blob/1799-indicator-optimisation-fixes/packages/indicators/src/AnalyticsRepository.ts#L83). I believe this is related to [this question](https://github.com/beyondessential/tupaia/pull/2071#discussion_r577244511)